File: VB/DXBindingExample/ViewModel/BasicExpressionsViewModel.vb
Imports DevExpress.Mvvm.DataAnnotations

Namespace DXBindingExample.ViewModel
    <POCOViewModel>
    Public Class BasicExpressionsViewModel
        Inherits BaseViewModel

        Public Overrides ReadOnly Property Title() As Object
            Get
                Return "Basic Expressions"
            End Get
        End Property
        Public Overridable Property IntValue() As Integer
        Public Overridable Property DoubleValue() As Double
        Public Overridable Property StringValue() As String
        Public Overridable Property BooleanValue() As Boolean
        Public Sub New()
            IntValue = 5
            DoubleValue = 6.1
            StringValue = "String"
        End Sub
    End Class
End Namespace

File: CS/DXBindingExample/ViewModel/BasicExpressionsViewModel.cs
using DevExpress.Mvvm.DataAnnotations;

namespace DXBindingExample.ViewModel {
    [POCOViewModel]
    public class BasicExpressionsViewModel : BaseViewModel {
        public override object Title { get { return "Basic Expressions"; } }
        public virtual int IntValue { get; set; }
        public virtual double DoubleValue { get; set; }
        public virtual string StringValue { get; set; }
        public virtual bool BooleanValue { get; set; }
        public BasicExpressionsViewModel() {
            IntValue = 5;
            DoubleValue = 6.1;
            StringValue = "String";
        }
    }
}
